File: permissionlib/src/main/java/com/benyanyi/permissionlib/PermissionDialogOnClick.java
package com.benyanyi.permissionlib;

import android.content.DialogInterface;

/**
 * @author YanYi
 * @date 2019/5/24 14:20
 * @email ben@yanyi.red
 * @overview
 */
interface PermissionDialogOnClick {
    /**
     * 点击
     *
     * @param failurePermissions
     */
    void onClick(DialogInterface dialog, String[] failurePermissions);
}

File: permission-kt/src/main/java/com/benyanyi/permission/kt/info/PermissionDialogOnClick.kt
package com.benyanyi.permission.kt.info

import android.content.DialogInterface

/**
 * @author YanYi
 * @date 2019/5/24 14:20
 * @email ben@yanyi.red
 * @overview
 */
interface PermissionDialogOnClick {
    /**
     * 点击
     *
     * @param failurePermissions
     */
    fun onClick(dialog: DialogInterface, failurePermissions: Array<String>)
}